pkg/cri/config/config_windows.go

```diff
@@ -40,7 +58,14 @@ func DefaultConfig() PluginConfig {
 			NoPivot:            false,
 			Runtimes: map[string]Runtime{
 				"runhcs-wcow-process": {
-					Type: "io.containerd.runhcs.v1",
+					Type:                 "io.containerd.runhcs.v1",
+					ContainerAnnotations: []string{"io.microsoft.container.*"},
+				},
+				"runhcs-wcow-hypervisor": {
+					Type:                 "io.containerd.runhcs.v1",
+					PodAnnotations:       []string{"io.microsoft.virtualmachine.*"},
+					ContainerAnnotations: []string{"io.microsoft.container.*"},
+					Options:              tree.ToMap(),
```

Comment:
`Options` is just a `map[string]interface{}`, do we need to round-trip through TOML, or can we just use a map literal here directly?

Reply:
No, the reason was mainly for the ease of including the comments which I assume was the unix files logic as well

Reply:
I see, per #5302 the interesting piece is that it was done this way specifically to show the comments in the output from `containerd config default`. I think that's valid, but preferably we would have a way to represent this more directly. How do comments get represented when `ToMap` is called?

Reply:
Sorry my explanation pertaining to including the comments wasn't very clear. `ToMap` doesn't preserve the comments in the map[string]interface{}, I meant just to have the comments readable in the source here and "config like". The other PR shows that the comments in the source don't show up either on `./containerd config default`. Let me know if you'd prefer to just swap to map like you said and I can comment above the keys.

Related thought, maybe we should do something similar and define some more of our shim options here to show what's available. That can be a QOL follow up to this

Reply:
If it's not preserving comments, it seems we can get all the same benefits (as well as added safety) by simply doing something like:

```golang
Options: map[string]interface{} {
    // SandboxIsolation tells us if the sandbox is isolated.
    "SandboxIsolation": 1,
    // SandboxPlatform tells us the sandbox platform.
    "SandboxPlatform": "windows/amd64",
}
```

Reply:
Right, I'm changin

Reply:
Changed to a map, resolving